test/mld_uc.jl

```diff
@@ -264,15 +264,55 @@ end
 =#
 
 @testset "discrete load and shunt variables" begin
-    @testset "3-bus uc case" begin
-        result = PowerModelsMLD._run_mld_discrete_load(case5_mld_strg_uc, DCPPowerModel, cbc_solver)
+    @testset "5-bus discrete case" begin
+        result = PowerModelsMLD._run_mld_discrete_load(case5_mld_strg_s, DCPPowerModel, cbc_solver)
```

Comment:
If you make your network changes to `case5_mld_strg_uc` directly can we save the need to add a new test case?  If it makes the `case5_mld_strg_uc` case infeasible we need two cases otherwise, I think we get away with one case for testing discrete generators, loads and shunts.

Reply:
This does work.  The objective/load_delivered tests for uc have been updated to match the edited case5.